app/assets/stylesheets/components/_accordion.scss

```diff
@@ -0,0 +1,109 @@
+.app-c-accordion {
+  &.js-accordion-with-descriptions {
+    margin-bottom: $gutter;
+    border-bottom: 1px solid $border-colour;
+  }
+}
+
+.app-c-accordion__header {
+  position: relative;
+  padding-top: 14px;
+  padding-bottom: 12px;
+
+  .js-accordion-with-descriptions & {
+    border-top: 1px solid $border-colour;
+    cursor: pointer;
+
+    &:hover {
+      background: $grey-3;
+    }
+  }
+}
+
+.app-c-accordion__icon {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  height: 16px;
+  width: 16px;
+  margin-top: -8px;
+  background-image: image-url('icons-plus-minus.png'); // PNG fallback for SVG
+  background: image-url("icons-plus-minus.svg"), linear-gradient(transparent, transparent); // SVG sprite
+  background-repeat: no-repeat;
+  background-position: 0 -16px;
+
+  @include media (tablet) {
+    right: 15px;
+  }
+
+  .section-is-open & {
+    background-position: 0 0;
+  }
+}
+
+.app-c-accordion__title {
+  @include core-24;
```

Comment:
you could use `bold-24` and remove the line below.

Reply:
Much better. It's interesting that I didn't know that was an option - is there any kind of documentation on what mixins are available to us and when we can use them? If not, maybe something we should think about, would be invaluable for new starters.

Reply:
No real documentation around this.

Here's the file:
https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_typography.scss#L129